addons/l10n_fr_invoice_addr/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_fr_is_company_french = fields.Boolean(compute='_compute_l10n_fr_is_company_french')
+
+    @api.depends('company_id.country_code')
+    def _compute_l10n_fr_is_company_french(self):
+        self.ensure_one()
```

Comment:
since it's a compute it should be a loop with for move in self and then move.l10n_fr_is_company_french = ....